src/Hydrator.php

```diff
@@ -55,15 +55,19 @@ public function add($path)
             );
         }
 
-        $defaults = [];
-        foreach ($resolver->getRelations($target) as $targetRelation) {
-            $name = $targetRelation->getName();
-            $isOne = $targetRelation->isOne();
+        $relationLoader = function (Model $subject, string $relationName) use ($target) {
+            $relation = $this->query->getResolver()->getRelations($target)->get($relationName);
+
```

Comment:
Why a newline here, but not before the `return`?

Reply:
Good question. Personal style I'd say. :sweat_smile: But I won't change it here, because it will change with #55 anyway.

Reply:
Understoood.